mapper/cumulocity/c8y_translator_lib/src/json.rs

```diff
@@ -0,0 +1,275 @@
+//! A library to translate the ThinEdgeJson into C8yJson
+//! Takes thin_edge_json bytes and returns c8y json bytes
+//!
+//! ```
+//! use c8y_translator_lib::json::CumulocityJson;
+//! let single_value_thin_edge_json = r#"{
+//!        "time": "2020-06-22T17:03:14.000+02:00",
+//!        "temperature": 23,
+//!        "pressure": 220
+//!     }"#;
+//! let output = CumulocityJson::from_thin_edge_json(
+//!             &String::from(single_value_thin_edge_json).into_bytes());
+//! ```
+
+use crate::serializer;
+use chrono::prelude::*;
+use thin_edge_json::{
+    json::{ThinEdgeJson, ThinEdgeValue},
+    measurement::GroupedMeasurementVisitor,
+};
+
+#[derive(Debug, Eq, PartialEq)]
+pub struct CumulocityJson;
+
+impl CumulocityJson {
+    ///Convert from thinedgejson to c8y_json
+    pub fn from_thin_edge_json(
+        input: &[u8],
+    ) -> Result<Vec<u8>, serializer::C8yJsonSerializationError> {
+        let local_time_now: DateTime<Local> = Local::now();
+        let timestamp = local_time_now.with_timezone(local_time_now.offset());
```

Comment:
Is that right to generate the timestamp at the time of serialisation and not time of arrival? Is there any requirement when should it be generated?